src/screens/HomePage/HomePage.js

```diff
@@ -6,23 +6,21 @@ import EnsureSharingIdInUrl from 'components/EnsureSharingIdInUrl';
 import ShareModelBlock from 'components/ShareBlock/ShareModelBlock';
 import CriteriaExplanation from './CriteriaExplanation/CriteriaExplanation';
 import Announcements from './Announcements/Announcements';
-import { PartnerLogoGrid, PressLogoGrid } from 'components/LogoGrid/LogoGrid';
 import { useLocation } from 'react-router-dom';
+import PartnersSection from 'screens/HomePage/PartnersSection';
 import HomePageThermometer from 'screens/HomePage/HomePageThermometer';
 import { GlobalSelector } from 'components/MapSelectors/MapSelectors';
 import { useHistory } from 'react-router-dom';
 import useMediaQuery from '@material-ui/core/useMediaQuery';
 import { useTheme } from '@material-ui/core/styles';
 import {
   Content,
-  FeaturedHeader,
-  PartnerSection,
-  PartnerHeader,
   SearchBarThermometerWrapper,
   SectionWrapper,
+  BannerContainer,
 } from './HomePage.style';
 import { SelectorWrapper } from 'components/Header/HomePageHeader.style';
-
+import DonationBanner from 'components/Banner/DonationBanner';
```

Comment:
nit: Consider exporting `DonationBanner` in `components/Banner/index.ts` and import from `components/Banner` here as we do it with the `FeedbackBanner`.

The idea of the `index.ts` file in component folders is to differentiate public vs. private components in component folders. For example, in `components/Expore/index.ts` we export `ExploreChart`, but not `SingleLocationChart` or `MultipleLocationsChart`. In this case, it doesn't matter much, so feel free to ignore it!